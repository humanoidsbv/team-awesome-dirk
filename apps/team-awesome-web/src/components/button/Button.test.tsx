import React from "react";
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";
import { Button } from ".";

describe("rendering of button", () => {
  it("renders the button ", () => {
    const tree = renderer
      .create(
        <Button>
          <div />
        </Button>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
