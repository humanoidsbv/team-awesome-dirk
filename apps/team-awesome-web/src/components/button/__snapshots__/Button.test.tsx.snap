// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of button renders the button  1`] = `
<button
  className="sc-bcXHqe ecOiny"
  onClick={[Function]}
>
  <div />
</button>
`;
